pyftpdlib/test/__init__.py

```diff
@@ -37,7 +37,10 @@
     unittest.TestCase.assertRaisesRegex = unittest.TestCase.assertRaisesRegexp
 
 if os.name == 'posix':
-    import sendfile
+    try:
+        import sendfile
+    except ImportError:
+        sendfile = None
```

Comment:
This was on purpose. I want tests to fail (and notice it) if pysendfile is not installed. (and `make setup-dev-env` will take care of that)

Reply:
Thank you for explanation. Reverted my changes.